extensions/amp-video/0.1/video-cache.js

```diff
@@ -215,3 +234,18 @@ function getCacheUrlService(videoEl, ampdoc) {
     .installExtensionForDoc(ampdoc, 'amp-cache-url')
     .then(() => Services.cacheUrlServicePromiseForDoc(videoEl));
 }
+
+/**
+ * Returns `true` if the video's inline response should be used instead of
+ * issuing an XHR request.
+ * @param {!Element} videoEl
+ * @return {boolean}
+ */
+function shouldUseInlineVideoResponse(videoEl) {
+  // Monti only inlines the first video of the first web story page.
```

Comment:
"Google video cache"*, this is the internal name we haven't used in the codebase :))